### ISSUE TITLE
Adding style to <SanityImage /> causes breaks lqip preview position

### ISSUE THREAD

**Issue by heggemsnes** [coreyward/sanity-image#30]
Great stuff as always

I'm using style on the image to make hotspot work correctly when using object fit like this:

```typescript
 <SanityImage
      id={_ref}
      projectId={env.NEXT_PUBLIC_SANITY_PROJECT_ID}
      dataset={env.NEXT_PUBLIC_SANITY_DATASET}
      width={width ?? undefined}
      height={height ?? undefined}
      mode="cover"
      hotspot={hotspot ?? undefined}
      crop={crop ?? undefined}
      preview={lqip}
      alt={alt ?? ""}
      sizes={sizes}
      style={{
        objectPosition: hotspot
          ? `${hotspot?.x * 100}% ${hotspot?.y * 100}%`
          : "center",
      }}
      loading={eager ? "eager" : "lazy"}
      className={cn(className)}
    />
```

I think due to the spread of props in the ImageWithPreview file using the style like this overwrites the styles while the image is loading.

**Comment by coreyward** [comment]
If I'm understanding you, you're saying that when you pass the `style` prop it replaces the default styles that are set on the full-size image during loading, right? If so, this is something I can fix, but I won't have bandwidth to do so for at least a few days. In the meantime, the simplest way to work around this is to include the default styles shown below in styles you set on the page scoped to the `[data-loading]` selector. Unfortunately you cannot do this via `style` prop.

https://github.com/coreyward/sanity-image/blob/ebfbb6f7e8e1dacf47df07a39694814b5932cdec/src/ImageWithPreview.tsx#L43-L67

**Comment by heggemsnes** [comment]
Yes Corey, you understood my rambling issue perfectly. 

No stress and apologize for the bad issue, was a late night problem 🙃

**Comment by heggemsnes** [comment]
Hi again Corey, this latest fix kinda also breaks using the `style` prop in general on the loaded image. Perhaps we could introduce a `previewStyle` prop for styling the lqip and keep the regular `style` for the actual image?

**Comment by coreyward** [comment]
Thank you for pointing this out!  I goofed and had the `style` prop being completely stripped. Moving too quickly I guess. I just released [v0.1.9](https://github.com/coreyward/sanity-image/releases/tag/v0.1.9) which should hopefully resolve this for you—I've added test coverage to verify and hopefully avoid further regressions. 

If you still think there's merit for a separate prop let me know and we can discuss.
